src/interfaces/ICapAutomator.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity >=0.8.0;
+
+interface ICapAutomator {
```

Comment:
Why is this declared here rather than just import from `sparklend-cap-automator`?

Reply:
Removed interface file and updated import